frontend/src/components/Courses/RateCourse.jsx

```
@@ -34,29 +34,30 @@ const RateCourse = ({ courseId }) => {
+        const star = index + 1;
+        return (
+          <button
+            key={star}
+            disabled={loading}
+            onClick={() => submitRating(star)}
+            onMouseEnter={() => setHover(star)}
+            onMouseLeave={() => setHover(0)}
+            className={`text-2xl transition-transform transform hover:scale-125 ${
+              star <= (hover || rating) ? 'text-yellow-400' : 'text-gray-300'
+            }`}
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)